i3_quickterm/main.py

```diff
@@ -24,6 +24,7 @@
     "term": "urxvt",
     "history": "{$HOME}/.cache/i3/i3-quickterm.order",
     "ratio": 0.25,
+    "width": 0.5,
```

Comment:
I would prefer keeping the existing defaults: `"width": 1.0,`.

Looking back into it, the `ratio` parameter is ugly and unhelpful, we can probably rename it to `height` at this point